src/test/java/com/solvd/carinapractice/olx/WebOlxTest.java

```diff
@@ -0,0 +1,136 @@
+package com.solvd.carinapractice.olx;
+
+import com.solvd.carinapractice.olx.components.CardComponent;
+import com.solvd.carinapractice.olx.pages.*;
+import com.zebrunner.carina.core.AbstractTest;
+import com.zebrunner.carina.utils.R;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+import java.util.Random;
+
+public class WebOlxTest extends AbstractTest {
+
+    private static final Logger LOGGER = LogManager.getLogger(WebOlxTest.class);
+    private static final String UNREGISTERED_EMAIL = R.TESTDATA.get("unregisteredEmail");
+    private static final String PASSWORD = R.TESTDATA.get("password");
+    private static final String SEARCH_INPUT = "airpods";
+    private static final String BARTER = "Обмін";
+
+    @Test(description = "test-case 1")
```

Comment:
as you chose description, add please description of your case in every test.
or change it to testName